docusaurus/docs-internals/editor-controls/introduction.md

```diff
@@ -3,4 +3,34 @@ sidebar_position: 1
 ---
 # Introduction
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean condimentum scelerisque eros vitae varius. Aliquam a est vestibulum, egestas eros et, posuere mi. Donec laoreet efficitur ultricies. Proin tincidunt dapibus ante, a dignissim ipsum dignissim sed. Etiam ut risus non diam suscipit dignissim. Suspendisse luctus ligula in ante efficitur, in vehicula ante feugiat. Nullam sollicitudin elit sit amet faucibus rhoncus. Nullam libero ex, commodo id pellentesque in, egestas sed ex. Mauris luctus a metus at congue. Aliquam tincidunt facilisis nunc quis lacinia. Suspendisse sit amet semper libero, ac fringilla arcu. Vestibulum eget semper quam. Nullam vitae suscipit velit.
+`Brizy` Editor Controls are the various input fields and UI elements used to build an interface for customizing website settings. These controls allow users to adjust the design and functionality of their sites directly from the `Brizy` editor, providing a live preview of changes.
+
+`Brizy` includes a comprehensive range of controls out-of-the-box, utilized in blocks, elements, and settings panels to gather user input and create custom outputs based on user preferences.
+
+### Brizy Controls
+
+`Brizy` offers various controls for different purposes, enabling users to fine-tune their designs without any coding knowledge. These controls include:
+
+1. **Text Controls**: Used to customize text elements such as headings, paragraphs, and buttons. Users can change font size, color, alignment, and more.
+
+2. **Image Controls**: Allow users to upload, replace, and edit images. Options include cropping, resizing, and applying filters.
+
+3. **Color Picker**: Provides an intuitive way to select and apply colors to different elements, including backgrounds, borders, and text.
```

Comment:
need to specify also that we can change the opacity, pallete, spread for shadow, gradient for background etc.